addons/website_slides/static/src/js/slides_course_content_manager.js

```diff
@@ -0,0 +1,189 @@
+            slides.push(parseInt($(this).data('slideId')));
+        });
+        return slides;
+    },
+    /**
+     *
+     * @param {*} data
+     */
+    _prepareNewContentValues: function (data){
+        this.slide = data.slide;
+        this.channel = data.channel;
+        this.category = data.category;
+        this.category.slides = this.slide ? [this.slide] : [];
+        this.modulesToInstallString = data.modulesToInstallString;
+    },
+    /**
+     * @private
+     */
+    _updateEmptyFlags: function (){
+        var categories = this.$('.o_wslides_slide_list');
+        for (var i = 0; i < categories.length; i++){
+            var categoryID = $(categories[i]).data('categoryId');
+            var categorySlideCount = $(categories[i]).find('.o_wslides_slides_list_slide:not(.o_not_editable)').length;
+            if (categorySlideCount === 0){
+                this.$('.category-empty[data-category-id='+ categoryID +']').removeClass('d-none');
+            } else {
+                this.$('.category-empty[data-category-id='+ categoryID +']').addClass('d-none');
+            }
+        }
+    },
+    /**
+     * Change links href to fullscreen mode for SEO.
+     *
+     * Specifications demand that links are generated (xml) without the "fullscreen"
+     * parameter for SEO purposes.
+     *
+     * This method then adds the parameter as soon as the page is loaded.
+     *
+     * @private
+     */
+    _updateHref: function () {
+        this.$(".o_wslides_js_slides_list_slide_link").each(function (){
+            var href = $(this).attr('href');
+            var operator = href.indexOf('?') !== -1 ? '&' : '?';
+            $(this).attr('href', href + operator + "fullscreen=1");
+        });
+    },
+    //--------------------------------------------------------------------------
+    // Handler
+    //--------------------------------------------------------------------------
+    /***
+     *
+     * @private
+     * @param {*} ev
+     */
+    _onAppendNewContent: function (ev){
+        var self = this;
+        this._prepareNewContentValues(ev.data);
+        var elementToAppendTo = this._getElementToAppendNewContentTo(ev.data.category_created, this.category.id);
+        var template = ev.data.category_created ? 'website.slide.list.category.item' : 'website.slide.list.slide.item';
+        elementToAppendTo.append(QWeb.render(template, {widget: this}));
+        this._onResequenceSlides();
+        this._attachWidgetsToAddedContent(ev.data.category_created).then(function (){
+            self.dragAndDropWidget._bindSortable();
+            self.$('.created-category').removeClass('created-category');
+            self.$('.created-slide').removeClass('created-slide');
+            ev.data.onSuccess();
+        });
+    },
+    /***
+     * @private
+     * @param {*} ev
+     */
+    _onArchiveContent: function (ev){
+        this.$('.o_wslides_slides_list_slide[data-slide-id='+ev.data.slideId+']').remove();
```

Comment:
missing space between () and { and also surrounding the '+'
(+ check everywhere)